### DISCUSSION TITLE
exportMenu 이 정상적으로 안되는데 제가 설정을 잘못한건가요?

### DISCUSSION THREAD

**Discussion by jewon7377** [nhn/tui.chart#681] [Q&A]
안녕하세요.

현재 TOAST UI CHART를 적용해보고 있는데요.
exportMenu 부분에서 visible: false로 처리를 해도 오른쪽 상단의 export부분이 노출되고 있습니다.

뭐가잘못된것인지 문의드립니다.

**Comment by jung-han** [comment]
@jewon7377 

What chart did you use?
If you write options and data together, it will be helpful to check more.
(It would be good if you write a version as well.)

```js
const options = {
  exportMenu: {
      visible: false,
    },
}
```
